engine/runtime-with-instruments/src/test/scala/org/enso/interpreter/test/instrument/RuntimeServerTest.scala

```diff
@@ -193,15 +192,13 @@ class RuntimeServerTest
                 Api.ExpressionUpdate(
                   Main.idMainY,
                   Some(ConstantsGen.INTEGER),
-                  if (noPointer) None
```

Comment:
IDE relies on method pointers in its _step into the node_ functionality. Engine should not exclude method pointer info from the update messages